BuildingSync.xsd

```diff
@@ -2016,6 +2023,45 @@
     </xs:sequence>
     <xs:attribute name="ID" type="xs:ID" use="required"/>
   </xs:complexType>
+  <xs:complexType name="FileType">
+    <xs:annotation>
+      <xs:documentation>File attachment associated with the audit report for supplemental information other than the general XML fields</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="FileName" type="xs:string" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Name identifying the file other than the ID</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="FileDescription" type="xs:string" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Details about the file</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="FileContentType" type="auc:ContentType" minOccurs="0">
```

Comment:
I would not recommend constraining the MIME type to `auc:ContentType` because the finite list of enumerations will inevitably have gaps.

Reply:
Do you suggest we just leave an unconstrained string field for this? I'm thinking we could add as many enumeration as needed, and user could use `Other` before the list is updated to fill the gap.

Reply:
The string format could be constrained using a regular expression, e.g., https://stackoverflow.com/a/48046041.